pom.xml

```diff
@@ -281,6 +283,50 @@
             </execution>
           </executions>
         </plugin>
+        <plugin>
+          <groupId>org.owasp</groupId>
+          <artifactId>dependency-check-maven</artifactId>
+          <version>${dependency-check-maven.version}</version>
+          <dependencies>
+            <dependency>
+              <groupId>ddf.support</groupId>
+              <artifactId>support-owasp</artifactId>
+              <version>${ddf.support.version}</version>
+            </dependency>
+            <dependency>
+              <groupId>org.mariadb.jdbc</groupId>
+              <artifactId>mariadb-java-client</artifactId>
+              <version>${mariadb.version}</version>
+            </dependency>
+          </dependencies>
+          <configuration>
+            <!-- The following properties enable using a centralized nvd server -->
+            <autoUpdate>${owasp.autoUpdate}</autoUpdate>
+            <databaseDriverName>${owasp.database.driverName}</databaseDriverName>
```

Comment:
❔ Does the settings file have these owasp properties defined?

Reply:
Discussed this out-of-band. Conclusion: we don't need these properties because we're not running a local cache of the NVD database and the dependency-check plugin will use the public NVD database by default. The downsides are:
1. Performance: using the local DB would save some time. On my machine (with a cache), it takes ~1 minute, whereas CircleCI takes ~3 minutes for the scan
2. Availability: Occasionally the public NVD database is unresponsive and could cause the scan to fail. Using a local copy resolves that issue.

For now, I've removed the properties with the expectation that we'll use the public database. If either of the above concerns becomes an issue, we can revisit this then